### PR TITLE
feat: leaderboard page updates

### PR DESCRIPTION
**Pull request by juniusfree**
## :pushpin: References

- **Wonder issue:** https://app.wonderverse.xyz/organization/wonderverse/boards?task=79972554171744534&entity=task
- **Related pull-requests:** n/a

## :blue_book: Description

1. Leaderboard page updates
2. Page headers updates
3. DatePicker range updates

## :movie_camera: Screenshot or Video

https://www.loom.com/share/53fd419893154d8d91ec2cd28187087c

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/components/organization/wrapper/BoardPageHeader.tsx [hunk 1]
```diff
@@ -207,14 +209,12 @@ function BoardPageHeader(props) {
...
         <TokenHeader>
-          <HeaderMainBlock>
-            <HeaderTopLeftContainer>
-              <HeaderTitle>{ENTITIES_DISPLAY_LABEL_MAP[entity?.toString()] || headerTitle}</HeaderTitle>
-              <PrivacyContainer>
-                <PrivacyText>{orgData?.privacyLevel !== PRIVACY_LEVEL.public ? 'Private' : 'Public'}</PrivacyText>
-              </PrivacyContainer>
-            </HeaderTopLeftContainer>
-          </HeaderMainBlock>
+          <Grid container width="fit-content" alignItems="center" gap="8px">
```
Comment (terryli0095) [inline wondrous-app/components/organization/wrapper/BoardPageHeader.tsx:212]: doesnt changing this also change other pages?

File: wondrous-app/components/Common/HeaderAvatars.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      paddingRight="8px"
+      sx={{
+        '& > div': {
+          marginRight: '-8px',
+        },
+      }}
+    >
+      {usersData.slice(0, AVATAR_LIST_OVERFLOW_MAX).map((user) => (
+        <Tooltip key={user.username} title={user.username} placement="top">
+          <Box
```
Comment (Lamperoyge) [inline wondrous-app/components/Common/HeaderAvatars.tsx:38]: can we have this as a Link?

File: wondrous-app/components/Leaderboard/LeaderboardUserRowTasks.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  const tasks = contributorTasks?.slice(0, taskCount) || [];
+  const showMoreButton = tasks.length < contributorTasks.length;
+  return (
+    <Grid container item width="100%" borderRadius="0 0 6px 6px">
+      <Grid container item width="100%" display="flex" gap="14px" padding="14px">
+        {tasks.map((task) => {
+          const podName = task?.podName || task?.pod?.name;
+          const podColor = task?.podColor || task?.pod?.color || palette.grey78;
+          const privateTask = task?.title === PRIVATE_TASK_TITLE;
+          const handleOnClickTask = (e) => {
```
Comment (Lamperoyge) [inline wondrous-app/components/Leaderboard/LeaderboardUserRowTasks.tsx:27]: can we use Link component? the reason is so we can have 'Click to open in a new tab'

File: wondrous-app/components/organization/wrapper/BoardPageHeader.tsx (hunk 1 above)
Comment (juniusfree) [reply wondrous-app/components/organization/wrapper/BoardPageHeader.tsx:212]: Yes, I updated it to include the other pages. I asked Ben about it. https://www.figma.com/file/UQAMcM7koodIoq2OTID7tj?node-id=23453:294651#374439477

File: wondrous-app/components/Common/HeaderAvatars.tsx (hunk 2 above)
Comment (juniusfree) [reply wondrous-app/components/Common/HeaderAvatars.tsx]: Updated.

File: wondrous-app/components/Leaderboard/LeaderboardUserRowTasks.tsx (hunk 3 above)
Comment (juniusfree) [reply wondrous-app/components/Leaderboard/LeaderboardUserRowTasks.tsx:27]: Updated.

**Review by andywong418 — APPROVED** [review]

**Pull request merged**